pkg/sql/grant_revoke.go

```diff
@@ -58,6 +59,18 @@ func (p *planner) changePrivileges(
 	grantees tree.NameList,
 	changePrivilege func(*sqlbase.PrivilegeDescriptor, string),
 ) (planNode, error) {
+	//check whether grantees exists
```

Comment:
Sorry to be so picky, but could you add a space at the beginning of the comment.